kolibri/plugins/management/assets/src/device_management/views/manage-content-page/channels-grid.vue

```diff
@@ -68,7 +68,7 @@
         return '';
       },
       sortedChannels() {
-        return this.channelList.sort(channel => channel.name);
+        return this.channelList.slice().sort(channel => channel.name);
```

Comment:
To address `no-side-effects-in-computed-properties`

Reply:
Nice. What a smart rule!

Reply:
Yeah, that's pretty good static analysis right there!